contracts/mixnet/src/helpers.rs

```diff
@@ -65,9 +72,88 @@ pub(crate) fn scale_reward_by_uptime(
     Ok(uint128_to_decimal(scaled))
 }
 
+// Extracts the node identity and owner of a delegation from the bytes used as
+// key in the delegation buckets.
+fn extract_identity_and_owner(bytes: Vec<u8>) -> StdResult<(Addr, IdentityKey)> {
```

Comment:
This function seems like some unnecessary hack - is there really no way around this?

Reply:
Answered above